packages/orchestration/src/examples/sendAnywhereFlows.js

```diff
@@ -0,0 +1,59 @@
+import { M, mustMatch } from '@endo/patterns';
+
+/**
+ * @import {Orchestrator, OrchestrationAccount} from '../types.js';
+ */
+
+const { entries } = Object;
+
+// in guest file (the orchestration functions)
+// the second argument is all the endowments provided
+
+export const orchestrationFns = harden({
+  /**
+   * @param {Orchestrator} orch
+   * @param {object} ctx
+   * @param {{ account: OrchestrationAccount<any> }} ctx.contractState
+   * @param {any} ctx.localTransfer
+   * @param {any} ctx.findBrandInVBank
+   * @param {ZCFSeat} seat
+   * @param {{ chainName: string; destAddr: string }} offerArgs
+   */
+  async sendIt(
+    orch,
+    { contractState, localTransfer, findBrandInVBank },
+    seat,
```

Comment:
Is there anything we can do to reduce boilerplate around this? (`orch`, `ctx`, `seat` params). The bulk of the noise in [kitchen-sink.contract.js](https://github.com/Agoric/agoric-sdk/blob/21742bd3d1f6c1ec6693b2378c6eb9efa7ee9999/packages/orchestration/src/examples/kitchen-sink.contract.js) seems to be from these.

Maybe curry orch and ctx so `sendIt` takes the normal `OfferHandler` type?

Reply:
because sendIt is a flow (guest fn) and not a regular function I think that's asking for trouble